### ISSUE TITLE
Cannot find module 'load-script'

### ISSUE THREAD

**Issue by dyaa** [kadirahq/meteor-dochead#11]
@kadirahq After updating the package to Version 1.2.0 is seems that there's an error

```
While processing files with cosmos:browserify (for target web.browser):
   packages/kadira:dochead/package.browserify.js: Cannot find module 'load-script' from '/home/User/dir/meteor/packages/dochead/.npm/package'
```

How can i solve that, I cant even run the server.

**Comment by jvbianchi** [comment]
+1

**Comment by arunoda** [comment]
Weird.

Try to remove `.meteor/local` directory in your project and try again.

**Comment by janisskuja** [comment]
+1 (on Windows 10)

and I tried deleting .meteor/local

```
 While processing files with cosmos:browserify (for target web.browser):
[...]/client.browserify.js: Cannot find
 module 'classnames/dedupe' from
 '[...]\.npm\package'
 packages/kadira:dochead/package.browserify.js: Cannot find module
 'load-script' from
 '\[...]\packages\kadira_dochead\1.2.2\npm'
```

**Comment by dyaa** [comment]
Solved at #12
